examples/inventory-rolling-update/resources/main.tf

```diff
@@ -78,7 +78,7 @@ resource "upcloud_server_group" "webservers" {
 }
 
 module "load_balancer" {
-  source = "UpCloudLtd/basic-loadbalancer/upcloud"
+  source = "git::https://github.com/UpCloudLtd/terraform-upcloud-basic-loadbalancer.git?ref=chore/update-tf-provider"
```

Comment:
Remove branch specifier after https://github.com/UpCloudLtd/terraform-upcloud-basic-loadbalancer/pull/2 is merged